blob/blob_test.go

```diff
@@ -296,6 +296,16 @@ func TestURLMux(t *testing.T) {
 	mux.RegisterBucket("foo", fake)
 	mux.RegisterBucket("err", fake)
 
+	if diff := cmp.Diff(mux.BucketSchemes(), []string{"err", "foo"}); diff != "" {
```

Comment:
make `[]string{"err", "foo"}` a local var `schemes` or some such and refer `foo` and `err` using `schemes[i]`.

Reply:
IMHO I don't think that makes the test clearer. The schemes are hard-coded in the URLs below too.

Reply:
I was thinking more about not needing to change a bunch of places when we do make change, but it's unlike to change since these are test files so I'm fine with it.